BLSQ/iaso
# IA-2358 : Add shortcut to submissions on the submission comparison table

Adding a link on the submission title from Compare submission page to redirect to the submission details.

Related JIRA tickets : [IA-2358](https://bluesquare.atlassian.net/browse/IA-2358)

## Self proofreading checklist


## How to test

- Go on the **Forms** menu and click on  **Submissions** link;
- From the Submissions list page, select 2 or more submissions to compare and click the Compare  icon;
- Click on any eye icon linked to submission title (e.g: Submission - 10297).

You should be redirected to the submission details page  


## Print screen / video


https://github.com/BLSQ/iaso/assets/6383219/8e2b5c79-b32e-4f71-abc9-dbf834cbbaf5




[IA-2358]: https://bluesquare.atlassian.net/browse/IA-2358?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
